src/renderers/webxr/WebXRController.js

```diff
@@ -269,6 +232,51 @@ class WebXRController {
 
 			}
 
+			if ( targetRay !== null ) {
+
+				inputPose = frame.getPose( inputSource.targetRaySpace, referenceSpace );
+
+				// Some runtimes (namely Vive Cosmos with Vive OpenXR Runtime) have only grip space and ray space is equal to it
+				if ( inputPose === null && gripPose !== null ) {
+
+					inputPose = gripPose;
+
+				}
```

Comment:
@toji Do you think this is the right way to handle this. Is the WebXR spec okay with this?

Reply:
This sounds like an error in the runtime, it really should be reporting a separate target ray pose. That said, this doesn't seem like a terrible compromise for cases when the runtime is simply broken. Hopefully that doesn't happen too often.

Not critical, but if you wanted to get fancy you'd probably want to pitch the inputPose down about 20deg or so from the grip pose to make it a little more natural to point with. (Otherwise it feels like you're trying to point at stuff with a sword.)